dist/images/Dockerfile.fedora

```diff
@@ -8,48 +8,53 @@
 # This is for a development build where the ovn-kubernetes utilities
 # are built locally and included in the image (instead of the rpm)
 #
-# At present this uses fedora:28
-# which has the needed ovs/ovn rpms at 2.8 or higher
-#
-# So this file will change over time.
 
 FROM fedora:29
 
 USER root
 
 ENV PYTHONDONTWRITEBYTECODE yes
 
+# install needed rpms - openvswitch must be 2.10.4 or higher
+# install selinux-policy first to avoid a race
+RUN dnf install --refresh -y  \
+	selinux-policy
```

Comment:
This is specific to RHEL. Fedora does not have the `openvswitch-selinux-extra-policy` package.